hardware-types/pcb/rfq.json

```
@@ -0,0 +1,328 @@
+                                    "type": "string"
+                                },
+                                "SolderBall": {
+                                    "title": "Solder Ball",
+                                    "default": "",
+                                    "type": "string"
+                                },
+                                "Position": {
+                                    "title": "Position",
+                                    "default": "",
+                                    "enum": [
+                                        "top",
+                                        "bottom",
+                                        "top & bottom"
+                                    ],
+                                    "type": "string"
+                                }
+                            },
+                            "additionalProperties": false
+                        },
+                        "PCBAssembly": {
+                            "type": "object",
+                            "title": "PCB Assembly",
+                            "required": [
+                                "NumUniqueParts",
+                                "NumPlacements",
+                                "NumThruHole",
+                                "NumSMT"
+                            ],
+                            "properties": {
+                                "NumUniqueParts": {
+                                    "default": 0,
+                                    "title": "Number of unique parts",
+                                    "type": "integer"
+                                },
+                                "NumPlacements": {
+                                    "default": 0,
+                                    "title": "Number of Unique placements",
+                                    "type": "integer"
+                                },
+                                "NumThruHole": {
+                                    "default": 0,
+                                    "title": "Number of through hole components",
+                                    "type": "integer"
+                                },
+                                "NumSMT": {
+                                    "title": "Number of SMT components",
+                                    "default": 0,
+                                    "type": "integer"
+                                }
+                            },
+                            "additionalProperties": false
+                        },
+                        "Other": {
+                            "type": "object",
+                            "title": "Other",
+                            "properties": {
+                                "CopperWeight": {
+                                    "title": "Copper Weight (oz)",
+                                    "default": 0.0,
+                                    "type": "number"
```

```suggestion
"title": "Copper Weight"
"default": 0,
"minimum": 0,
"multipleOf": 34.79,
"type": "number"
```

@ntzovanis, we work in SI but should we put just the copper weight in SI or stick with the imperial Oz for this?

---

@stordopoulos are these values above in SI or Oz?

I think we stick to Oz if thats what manufacturers tend to use, its what designers will be more familiar with. Once we have this change committed this PR can probably be merged?

---

@rcooke-warwick, Copper is something I'm conflicted about. Most sites from the US and China have Copper in "Oz" but EU-based sites have it in um - I think one uses mils, too. I think @ntzovanis, @zisis00, and @MTolba should offer their insights on this. As they are the ones that will be using the form.

If Altium, or whatever other software we use, uses Oz as the default it makes sense to use that as the default at well.

---

I updated it based on the values you suggested here